docs/usage/interpreter/interpreter_binding_mode.md

```diff
@@ -25,4 +25,97 @@ limitations under the License.
 
 ## Overview
 
+<center><img src="{{BASE_PATH}}/assets/themes/zeppelin/img/docs-img/interpreter_binding_per_note_user.png" height="70%" width="70%"></center>
+<center><img src="{{BASE_PATH}}/assets/themes/zeppelin/img/docs-img/interpreter_binding_scoped_isolated.png" height="70%" width="70%"></center>
 
+<br/>
+
+Interpreter is a JVM process that communicates with Zeppelin daemon using thrift. 
+Each Interpreter process can have a interpreter group, and each interpreter instance belongs to this interpreter group.
+(See [here](../../development/writing_zeppelin_interpreter.html) to understand more about its internal structure.) 
+
+Zeppelin provides 3 different modes to run interpreter process: **shared**, **scoped** and **isolated**.   
+Also, user can specify the scope of these mode as well: **per user** or **per note**.  
+These 3 modes give flexibility to fit Zeppelin into any type of use cases.
+
+In this documentation, we mainly discuss the combination of **per note** mode with **shared**, **scoped** and **isolated** modes for explanation. 
```

Comment:
In this documentation, we mainly discuss the **per note** scope in combination with the **shared**, **scoped** and **isolated** modes.